tests/conftest.py

```diff
@@ -66,7 +66,7 @@ def is_valid(self):
     __mapper_args__ = {"concrete": True}
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture()
```

Comment:
Does this cause a significant slow down in the unit test suite?

Reply:
I did not experience a slow down, so it should be insignificant.